cypress/pageObject/Patients/PatientEncounter.ts

```
@@ -27,6 +27,28 @@ export class PatientEncounter {
     return this;
   }
 
+  openSettingsTab() {
+    cy.get('[data-sidebar="content"]').contains("Settings").click();
+    return this;
+  }
+
+ openNotesTab() {
```

ESLint

[error] Insert `·`

(prettier/prettier)